test/test_field.jl

```diff
@@ -218,6 +218,8 @@ end
                     field_sz = test_field_sizes[t]
                     parent_sz = test_parent_sizes[t]
                     f_view = view(f, indices...)
+                    f_sliced = Field(f, indices)
+                    f2_sliced = Field(f^2, indices)

```

Comment:
There's no other test for abstract operations in this file... should we put this in `test_abstract_operations.jl`? The other test is perfect.

Reply:
Or actually I think `test_computed_field.jl` is the ideal place to test this.

Reply:
Yeah, move those around however you think is necessary. I never know what the proper place for tests are; that was just by best guess

Reply:
It's subjective!